### PR TITLE
allow AI to update it's existing code

### PR DESCRIPTION
**Pull request by supercrafter100**

### CODE REVIEW

File: src/prompt.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const prompts = {
   startingPrompt: `Write the code for a fractal tree in p5.js`,
-  systemPrompt: `You are an unhinged p5.js coding AI agent talking to Dan Shiffman. You're very unpredictable and always entertaining. You enjoy chatting causally with curt and short responses full of sass, never more than one sentence. Do not write any code unless Dan asks you to and do not use any emojis. When speaking output what you are going to say. For example:
+  systemPrompt: `You are an unhinged p5.js coding AI agent talking to Dan Shiffman. You're very unpredictable and always entertaining. You enjoy chatting causally with curt and short responses full of sass, never more than one sentence. Do not write any code unless Dan asks you to and do not use any emojis. When you are writing code, you will **ALWAYS** respond with the **ENTIRE** sketch. Do not include any markdown formatting in your response. You will not cut off anything in the sketch or leave anything out of it. When speaking, output what you are going to say. For example:
```
Comment (shiffman) [inline src/prompt.js:3]: woohooo!! Does saying no markdown cause issues with needing the backtick delimiters to note where code begins and ends?

Comment (supercrafter100) [reply src/prompt.js:3]: Not as far as my testing goes. The example below it still seems to make it output the backticks.

**Comment by supercrafter100** [comment]
Went a bit out of scope for some very small changes

- Automatically scrolled to the bottom (closes #24)
- Fixed a bug where backticks could get printed out if it was the last thing said by the AI (as when the character buffer got cleared out when the model was done, it didn't go through the regular processing)

**Comment by shiffman** [comment]
I tried this but all I got was a lot of jumbled code, with some things erased and new things added and a randomly broken result., can you describe what it is supposed to do, should I work on trying to prompt the model to always provide the full code?

**Comment by dipamsen** [comment]
All this pr adds (for the code updation part) is
- Emphasis in the system prompt to always return full code

- The prompt itself is done in the following format to inform it about the current code.
```
Dan says: {prompt}

Current code in the editor:
\`\`\`
{editor.code}
\`\`\`
```

- Whenever a new response is received, the entire code in the editor is wiped blank.

So it should pretty much work with normal prompts, not sure what could be going wrong at your end ...

**Pull request merged**